src/index.ts

```diff
@@ -0,0 +1,17 @@
+export class Challenge
+{
+    public buzz(i: number)
```

Comment:
Generally we prefer to name function parameters with more intuitive name. So please avoid `n`, `i`, `c` and prefer `entry`, `count`, `peers`, etc...